reporter-web-app/src/models/WebAppOrtResult.js

```diff
@@ -393,6 +393,18 @@ class WebAppOrtResult {
         return this.#repository;
     }
 
+    get repositoryConfiguration() {
+        return this.#repositoryConfiguration;
+    }
+
+    get ruleViolations() {
+        return this.#ruleViolations;
+    }
+
```

Comment:
I think we sometimes do this in the code base already.
But I always wonder why, e.g. wouldn't it make more sense to group closely related thing together?
I'm aware that the place in the alphabet can be a relation as well, but it usually starts to conflict with other (more important types of relations) at some point. I'm ok If you keep this commit but wanted to highlight the drawbacks at least.

Reply:
I am sorting alphabetically as that how it shown to me in it in my browser console / dev-utils and found out that organizing the code in this way helps me during development.